integration/heapster_api_test.go

```diff
@@ -174,7 +175,11 @@ func buildAndPushDockerImages(fm kubeFramework, zone string) error {
 	}
 	hostnames := []string{}
 	for _, node := range nodes {
-		hostnames = append(hostnames, strings.Split(node, ".")[0])
+		nodeHost := strings.Split(node, ".")[0]
+		if *testUser != "" {
+			nodeHost = fmt.Sprintf("jenkins@%s", nodeHost)
```

Comment:
Should it be reference or not &_@ I wrote too much python and forgot how go works...